src/zenml/cli/server.py

```diff
@@ -259,7 +259,8 @@ def down() -> None:
     deployer = ServerDeployer()
     deployer.remove_server(server.config.name)
 
-    GlobalConfiguration()
+    gc = GlobalConfiguration()
+    gc.set_default_store()
```

Comment:
Although obvious somehow, this feels like magic. Might it be worth adding another message to say that the current stack has been set to the default stack? I've certainly been burned by running a pipeline with the wrong stack in the past.

Reply:
> that the current stack has been set to the default stack

It doesn't do that though, I just tested manually. The logic of this function is quite complex but AFAIK this only tells ZenML to revert back to the default SQLZenStore, which is used in this function to disconnect from the local server in case it was already dead and couldn't be shut down.